### PR TITLE
Move the contract address to env

### PR DESCRIPTION
**Pull request by jakzaizzat**
To avoid any regression for contract address

Checklist for production
Add 3 env variables to vercel
1. `NEXT_PUBLIC_PREMIUM_MEMBERSHIP_CONTRACT_ADDRESS`
2. `NEXT_PUBLIC_GENERAL_MEMBERSHIP_CONTRACT_ADDRESS`
3. `NEXT_PUBLIC_GALLERY_MEMENTOS_CONTRACT_ADDRESS`

### CODE REVIEW

File: src/hooks/useContract.ts [hunk 1]
```diff
@@ -3,14 +3,19 @@ import { JsonRpcSigner, Web3Provider } from '@ethersproject/providers';
...
 import { network } from 'connectors/index';
 import { useActiveWeb3React } from './useWeb3';
 
-export const PREMIUM_MEMBERSHIP_CONTRACT_ADDRESS = '0xe01569ca9b39E55Bc7C0dFa09F05fa15CB4C7698';
-export const GENERAL_MEMBERSHIP_CONRTACT_ADDRESS = '0xe3d0fe9b7e0b951663267a3ed1e6577f6f79757e';
-
-export const GALLERY_MEMORABILIA_CONTRACT_ADDRESS = '0x35ae1f85389D2bbd48113909DA6182304952f809';
+export const PREMIUM_MEMBERSHIP_CONTRACT_ADDRESS =
+  process.env.NEXT_PUBLIC_PREMIUM_MEMBERSHIP_CONTRACT_ADDRESS ||
+  '0xe01569ca9b39E55Bc7C0dFa09F05fa15CB4C7698';
```
Comment (jakzaizzat) [inline src/hooks/useContract.ts:12]: Lmk if we wanna remove the fallback contract address

Comment (kaitoo1) [reply src/hooks/useContract.ts:12]: but otherwise PR looks good!!

Comment (Robinnnnn) [reply src/hooks/useContract.ts:12]: +1, thank you jakz!!

File: src/hooks/useContract.ts [hunk 2]
```diff
@@ -53,13 +55,14 @@ function useContract(address: string, abi: any) {
 }
 
 export function usePremiumMembershipCardContract() {
-  return useContract(PREMIUM_MEMBERSHIP_CONTRACT_ADDRESS, PREMIUM_MEMBERSHIP_CONTRACT_ABI);
+  return useContract(PREMIUM_MEMBERSHIP_CONTRACT_ADDRESS || '', PREMIUM_MEMBERSHIP_CONTRACT_ABI);
 }
 
 export function useGeneralMembershipCardContract() {
-  return useContract(GENERAL_MEMBERSHIP_CONRTACT_ADDRESS, GENERAL_MEMBERSHIP_CONTRACT_ABI);
+  console.log(GENERAL_MEMBERSHIP_CONRTACT_ADDRESS);
```
Comment (kaitoo1) [inline src/hooks/useContract.ts:62]:
```suggestion
```

**Review by kaitoo1 — APPROVED** [review]

**Review by kaitoo1 — APPROVED** [review]

**Pull request merged**